webapp/src/plugin.jsx

```diff
@@ -44,6 +44,7 @@ function getTranslations(locale) {
 
 export default class DemoPlugin {
     initialize(registry, store) {
+        registry.registerTranslations(getTranslations);
```

Comment:
shouldn't `registerTranslation` reload components if they're already loaded? if this isn't happening then maybe we should actually be doing that instead rather than relying on the order? unless i'm missing something 🤔

Reply:
Yes it should, it is not a fix to the reload problem. We should move registration forward anyway, why reload translations at all in this case?